src/python/WMCore/Database/MongoDB.py

```diff
@@ -0,0 +1,156 @@
+from pymongo import MongoClient, errors, IndexModel
+
+
+class MongoDB(object):
+    """
+    A simple wrapper class for creating a connection to a MongoDB instance
+    """
+    def __init__(self,
+                 database=None,
+                 server=None,
+                 port=None,
+                 create=False,
+                 collections=[],
+                 testIndexes=False,
+                 logger=None):
+        """
+        :databases:   A list of databases to connect to
+        :create:      A flag to trigger a database creation (if missing) during
+                      object construction, together with collections if present.
+        :collections: A list of tuples describing collections with indexes -
+                      the first element is considered the collection name, all
+                      the rest elements are considered as indexes
+        :testIndexes: A flag to trigger index test and eventually to create them
+                      if missing (TODO)
+        """
+        # DONE:
+        #    Database: msoutput
+        #    To create two different collections for Relval and NonRelval
+        # DONE:
+        #    To read the configuration parameters (server, port) from service config
+        self.server = server # '127.0.0.1'
+        self.port = port # 8230
+        try:
+            self.client = MongoClient(self.server, self.port)
+            self.client.server_info()
+        except Exception as ex:
+            msg = "ERR: could not connect to MongoDB server: %s\n%s" % (self.server, str(ex))
```

Comment:
Fixed. I am leaving only one explicit `ERR:` prefix on line: [1]. Because there  we are not logging but just throwing an exception. So from this place here there is no way to know which or what would be the top level  object to catch that and how would it treat it (it eventually may or may not use logger.erroro() to log it etc.). 


[1]
https://github.com/dmwm/WMCore/pull/9691/commits/eb2edafab1639a248d1771a1a344e0317710355e#diff-4ec8a9000f12097e2fd4dedc005dc385R89